File: ProgramasPython/secao06/Exercicio06.py
hora = 10;
excedente = 20;
e = 0;

c = input("Informe o código do funcionário: ")
n = int(input("Insira o nº de horas trabalhadas: "))

if n > 50:
    
    e = (n - 50) * excedente;
    salario = (50 * hora) + e;
    print("Salário total: R${0:.2f}.".format(salario))
    print("Salário excedente: R${0:.2f}.".format(e))
    
else:
    
    salario = n * hora;
    print("Salário total: R${0:.2f}.".format(salario))
    print("Salário excedente: R${0:.2f}.".format(e))

File: ProgramasJava/src/br/com/geekuniversity/secao06/Exercicio06.java
package br.com.geekuniversity.secao06;

import java.util.*;

public class Exercicio06 {

	public static void main(String[] args) {
		
		int codigo;
		float hora, salario, excedente = 0, valor = 10;
		Scanner tela = new Scanner (System.in); 
		
		System.out.println("Informe o código do funcionário: ");
		codigo	=	tela.nextInt();
		
		System.out.println("Informe o total de horas trabalhadas: ");
		hora	=	tela.nextFloat();
		
		if(hora > 50) {
			
			excedente	=	((hora - 50) * 20);
			salario		=	((50 * valor) + excedente);
			System.out.printf("%nCódigo do funcionário consultado: %d",	codigo);
			System.out.printf("%nSalário a ser pago: R$%.2f",			salario);
			System.out.printf("%nSalário excedente a ser pago:R$%.2f",	excedente);
			
		} else {
			
			salario = (hora * valor);
			System.out.printf("%nCódigo do funcionário consultado: %d",	codigo);
			System.out.printf("%nSalário a ser pago: R$%.2f",			salario);
			System.out.printf("%nSalário excedente a ser pago:R$%.2f",	excedente);
			
		}
		
		tela.close();
		
	}

}
